CHANGELOG.md

```diff
@@ -2,13 +2,22 @@
 
 ## Unreleased
 
+### Added
+
+- `GDPR_API_URL_PATTERN` setting for configuring the GDPR API URL pattern.
+- `GDPR_API_MODEL_LOOKUP` setting for configuring how the GDPR model instance is found.
+- `GDPR_API_USER_PROVIDER` setting for configuring how to obtain a `User` instance from the GDPR model instance.
+- `GDPR_API_DELETER` setting for configuring how data deletion is performed.
+
 ### Changed
 
 - Respond with a `204 No Content` status code when no data is found for the requested profile.
 
 ### Removed
 
 - Support for Python 3.6
+- `helsinki_gdpr.views.DeletionNotAllowed` exception class. Raising that exception produced responses with
```

Comment:
Could maybe add a note (or an example) about how to migrate away from DeletionNotAllowed if a project has used it before?

Reply:
Added a note to the changelog.